go-controller/pkg/ovn/namespace.go

```diff
@@ -235,9 +259,16 @@ func (oc *Controller) createNamespaceLocked(ns string) *namespaceInfo {
 	oc.namespacesMutex.Lock()
 	defer oc.namespacesMutex.Unlock()
 
+	namespace, err := oc.watchFactory.GetNamespace(ns)
```

Comment:
you dont need to do any of this here from lines 262-266. This is just an initialization function. You can explicitly set annotations to "" if you want to make things clear, but I think it is defaulted to empty anyway on line 271.